src/modules/AutoinstScripts.rb

```diff
@@ -473,568 +239,71 @@ def typeString(type)
+          ""
+        else
+          SCR.Read(
+            path(".target.string"),
+            script.log_path
           )
-          params = []
         end
-        params = params.join(" ")
-        scriptName = Ops.get_string(s, "filename", "")
-        if scriptName == ""
-          t = URL.Parse(Ops.get_string(s, "location", ""))
-          scriptName = basename(Ops.get_string(t, "path", ""))
-          scriptName = type if scriptName == ""
-        end
-        scriptPath = ""
-        if type == "pre-scripts" || type == "postpartitioning-scripts"
-          scriptPath = Builtins.sformat(
-            "%1/%2/%3",
-            AutoinstConfig.tmpDir,
-            type,
-            scriptName
-          )
-          Builtins.y2milestone("Writing %1 script into %2", type, scriptPath)
-          if Ops.get_string(s, "location", "") != ""
-            Builtins.y2debug(
-              "getting script: %1",
-              Ops.get_string(s, "location", "")
-            )
-            if !GetURL(Ops.get_string(s, "location", ""), scriptPath)
-              Builtins.y2error(
-                "script %1 could not be retrieved",
-                Ops.get_string(s, "location", "")
-              )
-            end
-          else
-            SCR.Write(
-              path(".target.string"),
-              scriptPath,
-              Ops.get_string(s, "source", "echo Empty script!")
-            )
-          end
-        elsif type == "init-scripts"
-          scriptPath = Builtins.sformat(
-            "%1/%2",
-            AutoinstConfig.initscripts_dir,
-            scriptName
-          )
-          if Ops.get_string(s, "location", "") != ""
-            scriptPath = AutoinstConfig.destdir + scriptPath # bnc961320
-            Builtins.y2debug(
-              "getting script: %1",
-              Ops.get_string(s, "location", "")
-            )
-            if !GetURL(Ops.get_string(s, "location", ""), scriptPath)
-              Builtins.y2error(
-                "script %1 could not be retrieved",
-                Ops.get_string(s, "location", "")
-              )
-            end
-          else
-            SCR.Write(
-              path(".target.string"),
-              scriptPath,
-              Ops.get_string(s, "source", "echo Empty script!")
-            )
-          end
-          Builtins.y2milestone("Writing init script into %1", scriptPath)
-          # moved to 1st stage because of systemd
-          # Service::Enable("autoyast");
-        elsif type == "chroot-scripts"
-          toks = URL.Parse(Ops.get_string(s, "location", ""))
-          # special == true ---> The script has to be installed into /mnt
-          # because it will be called in a chroot environment.
-          # (bnc#889931)
-          # Add /mnt only if the script is getting via GetURL.
-          # In the other case SCR will do it via target setting.
-          # (bnc#897212)
-          #
-          # FIXME: Find out why "nfs" has a special behavior.
```

Comment:
The new code does not mention "nfs". Was it dropped on purpose?

Reply:
yes, there should be no reason to do it, so I drop it. Nfs has to be already mounted in this stage.